src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -156,12 +154,6 @@ protected Map<String, String> getFileSystemProperties(BatchSinkContext context)
     return properties;
   }
 
-  @Override
-  protected String getOutputDir(long logicalStartTime) {
-    this.outputPath = super.getOutputDir(logicalStartTime);
```

Comment:
this variable should be set in the prepareRun() method by calling getOutputDir(context)